examples/simple_reverse.coffee

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env coffee
+# This is a simple example of how to use the slack-client module in CoffeeScript. It creates a
+# bot that responds to all messages in all channels it is in with a reversed
+# string of the text received.
+#
+# To run, copy your token below, then:
+#  npm install
+#   cd examples
+#   coffee simple_reverse.coffeescript
+
+Slack = require '..'
+
+token = 'xoxb-YOUR-TOKEN-HERE' # Add a bot at https://my.slack.com/services/new/bot and copy the token here.
+autoReconnect = true
+autoMark = true
+
+slack = new Slack(token, autoReconnect, autoMark)
+
+slack.on 'open', ->
+  channels = []
+  groups = []
+  unreads = slack.getUnreadCount()
+  key
```

Comment:
You don't need this, CoffeeScript will handle the variable declaring for you.  This may actually throw a reference error at runtime, not sure though.